### PR TITLE
Weatherapi

### PR DESCRIPTION
**Pull request by nxrara**
- weather is forecasted for each trip based on date of trip
- weather is also forecasted for each trail (4 days weather forecast)

### CODE REVIEW

**Comment by barbayjuliette** [comment]
For the trails show page: 
- I think it would look better in its own separate section, like "getting there", "reviews".
- Could you add the temperature as well ? 
- Instead of thursday, friday, saturday, sunday, could you change to
**today**, friday, saturday, sunday, so that it is clearer

File: config/initializers/open-weather-api.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# API key for weather
+# config.api_key = "6157d0b45dbc65d57d7bcb4569934b10"
```
Comment (barbayjuliette) [inline config/initializers/open-weather-api.rb:2]: Delete the comment so that nobody can see your API key.

**Review by f-Xiaoxi — DISMISSED** [review]
everything seems to work fine

File: config/initializers/open-weather-api.rb (hunk 1 above)
Comment (barbayjuliette) [reply config/initializers/open-weather-api.rb:2]: Where is your key stored now? 
You should do the same set up as for cloudinary, to not add it on github.

**Review by barbayjuliette — CHANGES_REQUESTED** [review]
i added a few comments. 
Plus make sure API key hidden before merging

**Review by barbayjuliette — APPROVED** [review]

**Pull request merged**